app/components/modal-confirm-deactivate/template.hbs

```diff
@@ -0,0 +1,14 @@
+<h4>Are you sure you want to deactivate:</h4>
+
+<div style="margin: 30px 10px;">
+  <h3>{{originalModel.displayName}}</h3>
+</div>
+
+<div class="text-muted" style="font-size: 12px;">
+  ProTip: Hold the {{alternateLabel}} key while clicking Delete to bypass this confirmation.
```

Comment:
This should match the label on the button I clicked ("Deactivate" or "Stop Services")